glean-core/src/metrics/ping.rs

```diff
@@ -135,23 +135,32 @@ impl PingType {
         this
     }
 
-    pub(crate) fn name(&self) -> &str {
+    /// Get the name of this Ping
+    pub fn name(&self) -> &str {
         &self.0.name
     }
 
-    pub(crate) fn include_client_id(&self) -> bool {
+    /// Whether the client ID will be included in the assembled ping.
+    /// See PingType::new for further details.
```

Comment:
Since there aren't any additional details, you can omit the cross linking back to `PingType::new`, I think.